### ISSUE TITLE
Support for "global" reCAPTCHA (for China, etc)

### ISSUE THREAD

**Issue by rrjanbiah** [ampproject/amphtml#26114]
## Describe the new feature or change to an existing feature you'd like to see

According to Google's reCAPTCHA FAQ https://developers.google.com/recaptcha/docs/faq#can-i-use-recaptcha-globally

> Can I use reCAPTCHA globally?
> Yes, please use "www.recaptcha.net" in your code in circumstances when "www.google.com" is not accessible.
> 
> First, replace <script src="https://www.google.com/recaptcha/api.js" async defer></script> with 
> <script src="https://www.recaptcha.net/recaptcha/api.js" async defer></script>
>After that, apply the same to everywhere else that uses "www.google.com/recaptcha/" on your site.

But the same option is not available in AMP reCAPTCHA component. 

## Additional context

Many of our Chinese users are complaining that they're blocked by reCAPTCHA and we don't seems to be having any workaround when using AMP

**Comment by caroqliu** [comment]
This is a good FR, thanks for bringing this to our attention!

**Comment by rrjanbiah** [comment]
Bump! Any workarounds?

**Comment by cathyxz** [comment]
I'm not super familiar with this code, but it seems like the recaptcha url is set here: https://github.com/ampproject/amphtml/blob/ad22ac12c828232c6857667c70125307e6651df6/3p/recaptcha.js#L57

I can see a reasonable feature where we add an attribute called `global` (or something else) and then change the url to `www.recaptcha.net`. @ampproject/wg-ui-and-a11y thoughts?

In case the WGs don't have enough bandwidth to implement this anytime soon, @rrjanbiah would you be interested in contributing the feature?

**Comment by rrjanbiah** [comment]
@cathyxz 

> In case the WGs don't have enough bandwidth to implement this anytime soon, @rrjanbiah would you be interested in contributing the feature?

I'm not sure. But, let me know once you check with WGs so that I can check closer (BTW, who is WG?)

**Comment by cathyxz** [comment]
Oh! By WG, I'm referring to the working groups who are responsible for maintaining the library. In this case @ampproject/wg-ui-and-a11y. @caroqliu or @nainar can either of you comment on the prioritization of this feature?

**Comment by caroqliu** [comment]
We're currently onboarding a new member to the working group, and this looks like a good size feature for them to get started with. We should be able to get to it soon accordingly. :)
